src/content/docs/tutorial-infrastructure-monitoring/explore-host-data.mdx

```diff
@@ -0,0 +1,124 @@
+
+Let's look at these metrics graphs toggled between 3 hours ago and 6 hours ago.
+
+<SideBySide>
+    <Side>
+<img
+    title="Tiles and graphs set to a 3 hour time parameter"
+    alt="A screenshot displaying two metrics graphs and tiles from the past 6 hours."
+    src={metrics1}
+
+/>    </Side>
+    <Side>
+<img
+    title="Tiles and graphs set to a 6 hour time parameter"
+    alt="A screenshot displaying two metrics graphs and tiles from the past 6 hours."
+    src={metrics2}
+/>    
+</Side>
+</SideBySide>
+
+* If there's been an unexpected increase in load, the **Events** tile will display that behavior with either too many or too few expected events. When you compare between two time parameters, you can get a better sense of anomalous behavior.   
+* The **Alerts** tile displays the number of hosts that have reached a critical or warning threshold. It can indicate whether you've found the parameter where an incident first appeared. 
+* The timeseries in your metrics graphs can clue you into possible causes. For example, your time series at 3 hours might not display an obvious increase in disk utilization. Toggled to a 6 hour parameter, you can see approximately when behavior started to change.
+
+    </Step>
+    <Step>
+        ### Filter down to a set of hosts
+
+Once you've confirmed the nature of an incident, you can look at the summary table to find failing hosts. The summary table displays the top 100 hosts from the last 80 minutes, sorted by alert severity. The summary table is static, so it won't change dynamically with the time picker. By focusing on hosts health from the last 80 minutes, you can continue to build the story around a failure. 
+
+According to our tiles, we have an unexpected number of events and four alerting hosts. If you're looking at an environment with upward to 1,000 hosts, you'll want to filter down to the specific set that are failing. In this case, you might only be interested in the four alerting hosts, as they're likely responsible for the doubling of event data. 
+
+<img
+    title="Summary table displaying four alerting hosts and one warning host"
+    alt="A screenshot displaying the summary table with alerting hosts."
+    src={summaryTableAlerting}
+/>  
+
+Here are two examples of filtering down to a set of hosts based on attributes:  
+
+* You can start with a simple filter for any hosts with alert severity critical: `alertSeverity = CRITICAL`. 
+* Let's say you notice that these hosts belong to the same region. You can try filtering with `awsRegion = us-east1a` to compare performance across all hosts in that region. If something hasn't met an alert threshold yet, you can corroborate your findings against more data points. 
+
+In general, we recommend filtering by certain attributes to help you define what set of hosts are associated with an incident. When in doubt, you can always start with our suggested filters as well. 
```

Comment:
needs a screenshot for the filter bar to add some context here; I kept thinking there was a filter associated with the table since that's the image right above this section.